starknet_py/tests/e2e/deploy/deployer_test.py

```diff
@@ -96,3 +100,43 @@ async def test_constructor_arguments_contract_deploy(
         sum([1, 2, 3]),
         {"value": 12, "nested_struct": {"value": 99}},
     )
+
+
+@pytest.mark.asyncio
+async def test_if_udc_is_deployed(client):
+    class_hash = await client.get_class_hash_at(
+        contract_address=DEFAULT_DEPLOYER_ADDRESS
+    )
+
+    assert isinstance(class_hash, int)
+    assert class_hash != 0
+
+
+@pytest.mark.asyncio
+async def test_if_udc_has_expected_abi(gateway_client):
+    code = await gateway_client.get_code(contract_address=DEFAULT_DEPLOYER_ADDRESS)
+
+    assert code.abi == universal_deployer_abi
```

Comment:
This is not a test of our codebase, but rather assertion of the environment. I am sure pytest has something like `before_module` to run it there.

Reply:
https://github.com/software-mansion/starknet.py/pull/374/commits/93401336b78f2abb712d8875bbcec1c9af31dcd3